worker/compile_commands_template.json

```diff
@@ -4299,6 +4299,49 @@
         "directory": "PATH/out",
         "file": "../src/RTC/Consumer.cpp"
     },
+    {
+        "arguments": [
```

Comment:
Why this change in this file?

Reply:
I assumed I needed to add this in order to compile ActiveSpeakerObserver.cpp. If there is a cleaner way please let me know.

Reply:
See docs/Building.md. That's file is generated automatically :)

Reply:
This seems like a file for IDE support, but definitely doesn't participate in build.

Reply:
Thanks. I will check that and issue a separate PR, after the batch commit.

Reply:
Actually no need for that (unless you want to make `make tidy` command work in your host :)

Reply:
I meant to say another commit. What would you like me to do for this? I tried running **make bear**, but get the following error: 

`bear -o compile_commands_template.json /Applications/Xcode.app/Contents/Developer/usr/bin/make
Unrecognized parameter: "-o"`

Reply:
I tried **--output** with no success. I will see what is going on. Perhaps a version mismatch for bear on OS-X.

Reply:
bear 2.1.5 on Big Sur is failing to compile. I will look into it later. I'll address the other issues for now.

Reply:
I have bear 2.7 running and working. However, when running tidy I am getting a lot of warnings and errors. The error is it can not find wchar.h. 

`27547 warnings and 1 error generated.`

The resulting compile_commands.json looks correct, and is correctly formatted. I have committed the unformatted compile_commands_template.json for now.